Ogury/src/main/java/com/mopub/mobileads/OguryInitializer.java

```diff
@@ -55,9 +59,23 @@ public static void startOgurySDKIfNecessary(@NonNull Context context, @NonNull M
         startOgurySDK(context, assetKey);
     }
 
+    public static void updateConsent() {
+        if (!sInitialized) {
+            return;
+        }
+        PersonalInfoManager personalInfoManager = MoPub.getPersonalInformationManager();
```

Comment:
I think that MoPub recommends to use `MoPub.canCollectPersonalInformation();` to get user consent status

Reply:
Fixed in f06768509dc6cb8cd9304a8b6f0272eec00defc1

Reply:
Why don't you directly use the `canCollectPersonalInformation()` static method from the `MoPub` class? It will avoid to handle the `personalInfoManager` checks

Reply:
Because documentation says its nullable so I need to do a null check.

Reply:
All networks checks if GDPR applies before reading the value of canCollectPersonalInformation(). I just used the same structure.

Reply:
We must still validates if GDPR applies before transmitting the value. We do not want fake consent values for people that do not see the notice.

Fixed in 1edd0d62c3ef588c388b4bf23603546d079e079a.

Reply:
For me is not necessary because `canCollectPersonalInformation()` already handles if GDPR applies and it returns `true` if it does not apply, because in the no GDPR region we can collect and use data.
Also, note that the `gdprApplies()` can returns `null` as it is a `Boolean`. This case is also handled in the `canCollectPersonalInformation()` and it returns 'false' because we don't know is the GRPR applies.

Reply:
We do not want to collect a true for a user that did not see the consent. If the user moves to EU, he will need to be exposed to a consent, we must keep it in unknown state.
I will change to also not collect the value if we do not know if the gdpr applies.

Reply:
You didn't handle the case where gdprApplies() returns null ?

Reply:
Fixed in 00613e10458598f6c5ac0036b64191162e593aaf.